stacks/api/bucket/history/history.service.ts

```diff
@@ -19,6 +19,10 @@ export class HistoryService {
     this.collection = this.db.collection<History>("history");
   }
 
+  getSchema(bucketId: ObjectId) {
+    return this.db.collection<any>("buckets").findOne({_id: bucketId});
+  }
```

Comment:
Could you use BucketService directly?

Reply:
Why? Any methods in this service don't use another service directly and there is the comment line like `// We can not use BucketDataService as a direct dependency`